Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h

```diff
@@ -131,6 +131,7 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
   void PrintSelf(ostream& os, vtkIndent indent) override;
 
   virtual const char* GetIcon() {return "";};
+  virtual const char* GetAddIcon() {return "";};
```

Comment:
What happens if the user does not override the method? Do we display some generic markup icon?

Reply:
Here is what happens with a new markup with no icons specified:

![screenshot](https://user-images.githubusercontent.com/1978682/111210019-02ff2e80-85cd-11eb-8424-da69164ff201.png)